.github/workflows/ci.yml

```diff
@@ -8,7 +8,11 @@ env:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    container: quay.io/ovirt/vdsm-test-centos-8
+    strategy:
+      fail-fast: false
+      matrix:
+        distro: [centos-8, centos-9]
+    container: quay.io/ovirt/vdsm-test-${{matrix.distro}}
```

Comment:
In existing references to variables we use spaces around curly braces `${{ like.this }}` instead of `${{this}}` - let's have them in new code too.